src/voter.rs

```diff
@@ -272,6 +272,14 @@ impl<H, N, E: Environment<H, N>> VotingRound<H, N, E> where
 			trace!(target: "afg", "Got incoming message");
 			let SignedMessage { message, signature, id } = incoming;
 
+			if !self.env.is_equal_or_descendent_of(self.votes.base().0, message.target().0.clone()) {
+				trace!(target: "afg", "Ignoring message targeting {:?} lower than round base {:?}",
```

Comment:
in the future, I'm pretty sure we could actually report misbehavior on this if we had a stored commit from this voter from recently.